src/app/notification-view-list/notification-view-list.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from '../shared/user.service';
+import { CouchService } from '../shared/couchdb.service';
+
+@Component({
+  template: `
+    Your Notifications
```

Comment:
Is this notification message or just a text, if it is text to display then please add translations